client/lcd/lcd_test.go

```diff
@@ -124,19 +124,9 @@ func TestVersion(t *testing.T) {
 	res, body := Request(t, port, "GET", "/version", nil)
 	require.Equal(t, http.StatusOK, res.StatusCode, body)
 
-	reg, err := regexp.Compile(`\d+\.\d+\.\d+(-dev)?`)
```

Comment:
Why are we removing this testcase? We should still tag our versions in a uniform manner IMO.

Reply:
I'll update the test - though formats differ between tagged releases and develop builds

Reply:
ACKd and incorporated